### DISCUSSION TITLE
Ability to export/import goals

### DISCUSSION THREAD

**Discussion by karahi** [plausible/analytics#1187] [Anything Else]
I just moved a section of site to a subdomain, and this particular section of site had a few `trackEvent` calls in it.

So I created a new project for the subdomain, and now have to tediously re-add all my goals by hand, rather than selecting the relevant goals from the original domain and exporting them, followed by re-importing them into the subdomain.

I'm sure its not a suuuper common occurrence, but its a bit of a pain when it does end up being necessary.

**Comment by ellotheth** [comment]
I've got my staging and production environments separated into different Plausible sites so I can test new tracking setups before they go live. Batch import/export would be handy in my scenario as well!
